plugins/Sid/CMakeLists.txt

```diff
@@ -1,51 +1,14 @@
 INCLUDE(BuildPlugin)
 
-INCLUDE_DIRECTORIES(resid)
+if(NOT ${LMMS_HAVE_SID})
```

Comment:
This won't work if `LMMS_HAVE_SID` is empty, as the command will be `if(NOT)`. I would omit the expansion and use the variable name alone - the `if` command will expand it internally before negating it.

Reply:
> This won't work if LMMS_HAVE_SID is empty, as the command will be if(NOT). I would omit the expansion and use the variable name alone - the if command will expand it internally before negating it.

Yeah, I wasn't too sure if CMake could interpret variables as is using certain operators (i.e., I knew something like ``if(<variable>)`` could work, but not ``if(NOT <variable>)``).